ring/util_test.go

```diff
@@ -96,21 +96,14 @@ func TestGenerateTokensIgnoresOldTokens(t *testing.T) {
 	}
 }
 
-func TestWaitRingStabilityShouldReturnAsSoonAsMinStabilityIsReachedOnNoChanges(t *testing.T) {
-	t.Parallel()
-
-	const (
-		minStability = 2 * time.Second
-		maxWaiting   = 10 * time.Second
-	)
-
+func createStartingRing() *Ring {
 	// Init the ring.
 	ringDesc := &Desc{Ingesters: map[string]InstanceDesc{
-		"instance-1": {Addr: "127.0.0.1", State: ACTIVE, Timestamp: time.Now().Unix()},
-		"instance-2": {Addr: "127.0.0.2", State: PENDING, Timestamp: time.Now().Unix()},
-		"instance-3": {Addr: "127.0.0.3", State: JOINING, Timestamp: time.Now().Unix()},
-		"instance-4": {Addr: "127.0.0.4", State: LEAVING, Timestamp: time.Now().Unix()},
-		"instance-5": {Addr: "127.0.0.5", State: ACTIVE, Timestamp: time.Now().Unix()},
+		"instance-1": {Addr: "instance-1", State: ACTIVE, Timestamp: time.Now().Unix()},
```

Comment:
In practice, this field contains an IP address, so previous example was bit better, although in this test, it doesn't matter.

Reply:
fixed